test/util/validationUtils.test.js

```diff
@@ -388,20 +388,15 @@ describe('validateCareGapsParams', () => {
     }
   });
 
-  test('validateCareGapsParams throws error with both program and measure identification', async () => {
+  test('validateCareGapsParams  no longer throws error with both program and measure identification', async () => {
     const INVALID_REQ = {
       query: PROGRAM_AND_MEASURE_QUERY
     };
     try {
       validateCareGapsParams(INVALID_REQ.query);
-      throw new Error(
-        'validateCareGapsParams failed to throw an error when provided both program and measure identification'
-      );
     } catch (e) {
+      console.log(JSON.stringify(e));
       expect(e.statusCode).toEqual(501);
-      expect(e.issue[0].details.text).toEqual(
-        'Simultaneous program and measure identification (measureId/measureIdentifier/measureUrl) is not currently supported by the server.'
-      );
```

Comment:
This test isn't quite set up right. Right now it will pass if validateCare Gaps throws no error or a 501. What we want is to wrap the validateCareGaps call in an expect().toBeUndefined. We can actually get rid of the whole try/catch block, since if an error is thrown, the test will fail with a comprehensive message anyway.